src/components/BookListItem/BookListItem.tsx

```diff
@@ -11,17 +12,20 @@ interface BookProps {
   };
 }
 
-const BookListItem = ({ item }: BookProps) => {
+const BookListItem = ({ item: { id, title, author } }: BookProps) => {
+  const { role } = useContext(RoleContext);
+  const path = role === 'ADMINISTRATOR' ? `/dashboard/book/${id}` : `/book/${id}`;
```

Comment:
Can we have users role types somewhere in consts so we don't type it every time its error prone?